lib/index.ts

```diff
@@ -14,14 +14,14 @@ function getAllSwears(): Array<swearVariants> {
         [`calvaire`, `calvince`],
         [`mosus`],
         [`maudit`, `mautadit`, `maudine`],
-        [`sacrament`],
+        [`sacrament`, `saint-sacrament`],
         [`viarge`, `sainte-viarge`, `bout d'viarge`],
         [`cibouleau`],
         [`sacréfice`],
         [`cibole`, `cibolac`],
         [`enfant d'chienne`],
         [`verrat`],
-        [`marde`, `maudite marde`],
+        [`marde`, `maudite marde`, `mange-marde`],
```

Comment:
On n'entendrait pas plus souvent `mangeux d'marde`?

Reply:
Une [source bibliographique](https://youtu.be/1BdwuAt_5LM?t=20) à l'appui

Reply:
oui sans problème, désolé pour le spam, tu peux merger après le hacktoberfest si tu veux, j'aurais vraiment du prendre le temps et mettre toutes mes idées dans le même commit, my bad :)

Reply:
Pas de soucis! Ca ne me dérange pas de merger avant, mais j'éviterais de mettre le label à nouveau.

Veux-tu remplacer `mange-marde` par `mangeux d'marde` avant que je merge?

Reply:
Yes je change ça

Reply:
C'est vraiment priceless comme conversation de code review.

Reply:
j'avoue qu'on voit pas des titres de commit comme ça tous les jours